content/en/apps/reference/app-settings/transitions.md

```diff
@@ -436,6 +441,32 @@ Supported `events_types` are:
   }
 ```
 
+### Accept case reports
+
+Allow reporting about case centric workflows by
+- validating the report configuration documentation,
+- assigning the relevant place to the report if a registration with the given case_id exists,
+- clearing messages on the registration, and
+- generating response messages on the report.
+
+#### Example
+
+```json
+"registrations": [{
+  "form": "8"
```

Comment:
If the registration is included in the example, should it have `add_case`?

Reply:
Yes, this was a truncated example. I've added the event config as well to avoid confusion.